shipdata/ShipStack.cxx

```diff
@@ -418,8 +411,7 @@ Int_t ShipStack::GetCurrentParentTrackNumber() const
 TParticle* ShipStack::GetParticle(Int_t trackID) const
 {
   if (trackID < 0 || trackID >= fNParticles) {
-    fLogger->Info(MESSAGE_ORIGIN, "ShipStack: Particle index %i out of range. Max=%i",trackID,fNParticles);
-    Fatal("ShipStack::GetParticle", "Index out of range ");
+    LOG(INFO) << "ShipStack: Particle index %i out of range. Max=%i",trackID,fNParticles;
```

Comment:
Is it intentional that the `Fatal` line was removed here?

Reply:
yes. Looked to me strange to have an info message followed by fatal.

Reply:
Yes, it looks very strange...
Maybe we should make it an ERROR instead of INFO though? Not sure whether it makes sense to exit though...